javagen/src/main/java/com/azure/autorest/model/clientmodel/ProxyMethodExample.java

```diff
@@ -110,9 +102,8 @@ public Response(int statusCode, Object response) {
                 Map<String, Object> responseMap = (Map<String, Object>) response;
                 if (responseMap.containsKey("headers") && responseMap.get("headers") instanceof Map) {
                     Map<String, Object> headersMap = (Map<String, Object>) responseMap.get("headers");
-                    headersMap.forEach((header, value) -> {
-                        httpHeaders.add(HttpHeaderName.fromString(header), value.toString());
-                    });
+                    headersMap.forEach(
+                        (header, value) -> httpHeaders.add(HttpHeaderName.fromString(header), String.valueOf(value)));
```

Comment:
So the only logic change is here?

Reply:
Yeah, this is the meaningful logic change